overview.md

```diff
@@ -0,0 +1,41 @@
+# Overview
+
+How everything should look like after running through the tutorial:
+
+```mermaid
+
+flowchart LR
+    subgraph namespace: observability-backend
+        subgraph pod: collector
+            OC{OTel Collector}
+        end
+        subgraph pod: mimir
+            OC --metrics-->Mimir
+        end
+        subgraph pod: loki
+            OC --logs-->Loki
+        end
+        subgraph pod: tempo
+            OC --traces-->Tempo
+        end
+        subgraph pod: grafana
+            grafana-.->Mimir
```

Comment:
shouldn't the arrow point from the store to grafana? If we are denoting the data flow

Reply:
I was not really sure how to put the flows from grafana to mirmir,tempo,loki, because grafana is querying their APIs (it's a pull not a push, right?)

Reply:
I changed the arrows, I am not yet 100% happy with it, but perfect is the enemy of good